sale_medical_prescription/tests/test_medical_pharmacy.py

```diff
@@ -16,6 +16,7 @@ def setUp(self):
 
     def test_compute_verified_by_id(self):
         """ Test verified_by_id properly set to user """
+        self.pharmacy_4.is_verified = True
```

Comment:
I'm confused why these are all needed? The test was passing & you're editing data without any effect on the tests whatsoever. This means either:
* The change is frivolous
* The test is broken

Which is it?